TSRM/TSRM.c

```diff
@@ -746,14 +746,14 @@ TSRM_API size_t tsrm_get_ls_cache_tcb_offset(void)
 
 # ifdef __APPLE__
 	// Points to struct TLVDecriptor for _tsrm_ls_cache in macOS.
-	asm("adrp %0, #__tsrm_ls_cache@TLVPPAGE\n\t"
-	    "ldr %0, [%0, #__tsrm_ls_cache@TLVPPAGEOFF]"
-	     : "=r" (ret));
+	asm ("adrp %0, #__tsrm_ls_cache@TLVPPAGE\n\t"
+	     "ldr %0, [%0, #__tsrm_ls_cache@TLVPPAGEOFF]"
+	      : "=r" (ret));
 # else
-	asm("mov %0, xzr\n\t"
-	    "add %0, %0, #:tprel_hi12:_tsrm_ls_cache, lsl #12\n\t"
-	    "add %0, %0, #:tprel_lo12_nc:_tsrm_ls_cache"
-	     : "=r" (ret));
+	asm ("mov %0, xzr\n\t"
+	     "add %0, %0, #:tprel_hi12:_tsrm_ls_cache, lsl #12\n\t"
+	     "add %0, %0, #:tprel_lo12_nc:_tsrm_ls_cache"
+	      : "=r" (ret));
```

Comment:
These TSRM wite-space changes are not related to IS_SIGNED_32BIT

Reply:
Yes. Will revert the update on this file. Thanks.

Reply:
Yes. Will revert the update on this file. Thanks.